sqlalchemy-stubs/orm/session.pyi

```diff
@@ -45,6 +45,8 @@ class Session(_SessionClassMethods):
                  binds: Optional[Any] = ..., extension: Optional[Any] = ...,
                  info: Optional[Any] = ..., query_cls: Any = ...) -> None: ...
     connection_callable: Any = ...
+    def __enter__(self): ...
+    def __exit__(self, type, value, traceback): ...
```

Comment:
The parameter and return type annotations are missing.

Reply:
Same for this method.

Reply:
```suggestion
    def __exit__(self, type: Type[BaseException], value: BaseException, traceback: TracebackType) -> Optional[bool]: ...
```